resources/views/docs/events.blade.php

```diff
@@ -9,6 +9,7 @@
   * [Scoping To Components By Name](#scope-by-name) { .font-normal.text-sm.text-blue-800 }
   * [Scoping To Self](#scope-to-self) { .font-normal.text-sm.text-blue-800 }
 * [Listening For Events In JavaScript](#in-js) { .text-blue-800 }
+* [Dispatching Browser Events](#browser) { .text-blue-800}
```

Comment:
` { .text-blue-800 }`

Reply:
Apologies, I will correct the formatting!